Content.Server/GameObjects/Components/Items/rcdComponent.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using Content.Server.GameObjects.EntitySystems;
+using Content.Server.Interfaces;
+using Content.Shared.Maps;
+using Robust.Server.GameObjects.EntitySystems;
+using Robust.Server.Interfaces.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Components.Transform;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.GameObjects.Components;
+using Robust.Shared.Interfaces.Map;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Robust.Shared.Map;
+using Robust.Shared.Serialization;
+using Robust.Shared.Utility;
+
+namespace Content.Server.GameObjects.Components.Items
+{
+    [RegisterComponent]
+    public class RcdComponent : Component, IAfterAttack, IUse, IExamine
```

Comment:
Change to `RCDComponent`

Reply:
Actually C# standard is PascalCase for abbreviations greater than two characters. So `System.IO` but `HttpClient`, so this was correct.